WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -161,11 +163,11 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
     NSInteger settingsSection = [self.tableSections[section] intValue];
     switch (settingsSection) {
         case SiteSettingsSectionGeneral: {
-            if ([self.blog supports:BlogFeaturePushNotifications]) {
-                return SiteSettingsGeneralCount;
-            } else {
-                return SiteSettingsGeneralCount-1;
+            NSInteger rowsToHide = 0;
+            if (![self.blog supports:BlogFeatureWPComRESTAPI]) {
+                rowsToHide += 1;
```

Comment:
Maybe a small comment here to describe what we're hiding when it's a WP.com blog?